packages/cli/lang/en.lyaml

```diff
@@ -810,6 +813,7 @@ en:
           uploadComplete: "Uploading files to \"{{ dest }}\" in the Design Manager is complete"
         uploading: "Uploading files from \"{{ src }}\" to \"{{ dest }}\" in the Design Manager of account {{ accountId }}"
         notUploaded: "There was an error processing \"{{ src }}\". The file has not been uploaded."
+        overwriting: "Overwriting \"{{ path }}\" on account {{ accountId }}..."
```

Comment:
Some of the verbiage from `cli-lib`: https://github.com/HubSpot/cli-lib/blob/f2283d961c2a12e3e78793dc6991d0f47e34caba/lib/watch.js#L202-L221